features/step_definitions/content.rb

```diff
@@ -55,3 +55,33 @@
   }
   expect(page).to have_xpath("//h1[contains(text(),'#{I18n.t *translations[text]}')]")
 end
+
+
+Then(/^there should be a back button on all pages except the start page$/) do
```

Comment:
I think this should be an RSpec test so you can have a list of all relevant routes in your test ui.

Reply:
We thought about this, but this is all functionality, that we might test: What do you think is best: Only a Cucumber test, only a RSpec test or a Cucumber and a redundant RSpec test?

Reply:
Ideally, just an rspec test, this seems to granular for an rspec test.

Reply:
Do you mean "this seems too granular for a cucumber test"? Then I would copypaste all lines out of the step definitions, create a rspec with it and delete this test?